README.md

```diff
@@ -85,6 +85,9 @@ gProfiler can be run in a continuous mode, profiling periodically, using the `--
 Note that when using `--continuous` with `--output-dir`, a new file will be created during *each* sampling interval.
 Aggregations are only available when uploading to the Granulate Performance Studio.
 
+### Legacy profile upload mode
```

Comment:
This is currently a private option for our use so I don't think it deems documentation here